packages/mux-player/src/themes/classic/index.ts

```diff
@@ -0,0 +1,18 @@
+// @ts-ignore
+import theme from './classic.html';
+import { document } from '../../polyfills';
+import { MediaThemeElement } from 'media-chrome/dist/media-theme-element';
+import { window } from 'media-chrome/dist/utils/server-safe-globals.js';
```

Comment:
I believe this is causing all the typescript issues. We probably want to use the elements polyfill rather than importing media-chrome's server-safe-globals.